.gitignore

```
@@ -0,0 +1,19 @@
+# Ignore vi swapfiles
+*.swp
+.*.swp
+
+# Ignore Python cache directories
+__pycache__/
+
+# Ignore project coverage
+.coverage
+
+# Ignore tox working directory
+.tox/
+
+# Ignore "egg info" directories
+*.egg-info
```

I guess if we want to limit it to directories we could do this:

```suggestion
*.egg-info/
```

although I can't imagine any circumstances in which a _file_ matching `*.egg-info` would be part of the project so it probably doesn't matter :shrug:

---

Yep, and if someone does create a `.egg-info` _file_, we don't want it here either. :-)